pkg/kms/localkms/pubkey_writer.go

```diff
@@ -97,7 +98,8 @@ func write(w io.Writer, msg *tinkpb.Keyset) error {
 func writePubKey(w io.Writer, key *tinkpb.Keyset_Key) (bool, error) {
 	var marshaledRawPubKey []byte
 
-	// TODO add other key types than the ones below and other than ecdhAESPublicKeyTypeURL
+	// TODO add other key types than the ones below and other than nistPECDHKWPublicKeyTypeURL and
```

Comment:
Issue #?

Reply:
there's no specific issue # for this, BBS+ is the next new key type to be added in the kms, I'll remove this comment when BBS+ key type is added to the kms (and the same will be added when the Bitcoin key type is added later)